src/overlays/actors/ovl_Dm_Bal/z_dm_bal.h

```diff
@@ -2,17 +2,28 @@
 #define Z_DM_BAL_H
 
 #include "global.h"
+#include "objects/object_bal/object_bal.h"
 
 struct DmBal;
 
 typedef void (*DmBalActionFunc)(struct DmBal*, GlobalContext*);
 
 typedef struct DmBal {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x44];
-    /* 0x0188 */ DmBalActionFunc actionFunc;
-    /* 0x018C */ char unk_18C[0x1B0];
-} DmBal; // size = 0x33C
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ DmBalActionFunc actionFunc;
+    /* 0x18C */ f32 unk_18C;
+    /* 0x190 */ f32 unk_190; 
+    /* 0x194 */ f32 unk_194;
+    /* 0x198 */ s32 unk_198;
+    /* 0x19C */ Vec3s morphTable[OBJECT_BAL_LIMB_MAX];
+    /* 0x262 */ Vec3s joinTable[OBJECT_BAL_LIMB_MAX];
```

Comment:
```suggestion
    /* 0x19C */ Vec3s jointTable[OBJECT_BAL_LIMB_MAX];
    /* 0x262 */ Vec3s morphTable[OBJECT_BAL_LIMB_MAX];
```
Looks like the `jointTable` and `morphTable` are reversed. Also `jointTable` looks to be missing a `t`.